sdk/servicebus/Azure.Messaging.ServiceBus/src/Management/HttpRequestAndResponse.cs

```diff
@@ -273,7 +273,6 @@ public async Task<string> GetTokenAsync(string requestUri)
                 var token = await GetToken(request.Uri.ToUri()).ConfigureAwait(false);
                 request.Headers.Add("Authorization", token);
             }
-            request.Headers.Add("UserAgent", $"SERVICEBUS/{ManagementClientConstants.ApiVersion}(api-origin={ClientInfo.Framework};os={ClientInfo.Platform};version={ClientInfo.Version};product={ClientInfo.Product})");
```

Comment:
This is included automatically by the pipeline when diagnostics are enabled. It was also using the incorrect header here - "UserAgent" rather than "User-Agent"